### PR TITLE
Lock requirements

### PR DESCRIPTION
**Pull request by paulbovbel**
Lock requirements via catkin_virtualenv

### CODE REVIEW

File: aiorospy/CMakeLists.txt [hunk 1]
```diff
@@ -9,14 +9,15 @@ catkin_package()
 
 catkin_python_setup()
 
-install(FILES requirements.txt
+install(FILES requirements.in
```
Comment (ablakey) [inline aiorospy/CMakeLists.txt:12]: Is `.in` a standard or convention from somewhere?

File: aiorospy/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,11 @@
-aiounittest==1.2.1
-janus==0.4.0
+aiounittest==1.3.1        # via -r /home/pbovbel/locus_dev/src/aiorospy/aiorospy/requirements.in
```
Comment (ablakey) [inline aiorospy/requirements.txt:1]: To confirm, this whole file was generated by the new catkin_virtualenv tooling?

File: aiorospy/CMakeLists.txt (hunk 1 above)
Comment (paulbovbel) [reply aiorospy/CMakeLists.txt:12]: it is in pip-tools, but i now regret that decision.

I'm going to switch to `requirements.txt` and `Pipfile.lock`, which is standard-ish and supported by github's vulnerability scanner

Comment (paulbovbel) [reply aiorospy/CMakeLists.txt:12]: it's the default from `pip-tools` (https://github.com/locusrobotics/catkin_virtualenv/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R117).

One advantage of keeping the locked dependencies in `requirements.txt` is that it'll get scanned by tools such as github's vulnerability scanner.

File: aiorospy/requirements.txt (hunk 2 above)
Comment (paulbovbel) [reply aiorospy/requirements.txt:1]: yep

**Pull request merged**